packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -55,7 +55,7 @@ type AuthOptions = {
 
 type GetResolverParams<TSource, TContext, TArgs> = {
   operation: Operation
-  argsFromLink?: { [key: string]: string }
+  argsFromLink?: { [key: string]: any }
```

Comment:
So I had to make a change here.

Source: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#link-object

Reply:
right... I totally missed this, thanks for spotting this